dnf-behave-tests/dnf/check-update.feature

```diff
@@ -30,3 +30,15 @@ Given I use repository "dnf-ci-fedora-updates" with configuration
  When I execute dnf with args "upgrade"
  Then the exit code is 0
   And Transaction is empty
+
+
+@bz2101421
+Scenario: --security check-update doesn't show pkgs from resolved advisories (when obsoletes are involved)
+Given I use repository "check-update"
+  And I execute dnf with args "install A-1-1"
+ When I execute dnf with args "update --security"
+ Then the exit code is 0
+ And Transaction is empty
+ When I execute dnf with args "check-update --security"
+ Then the exit code is 0
+ And stdout is empty
```

Comment:
The test "almost" passes even on unpatched dnf. I mean, the output in the last step contains only "Last metadata expiration check: 0:00:01 ago on Po 8. srpna 2022, 15:26:55.". Is that alright?
As far as I understand the bugreport, it should contain some obsoleting packages, right?

Reply:
Hmm, the output from GHA check shows some obsoleting packages. Maybe something is wrong with my local environment.

Reply:
Oh but then maybe I should change it to:
```Gherkin
And stdout is
"""
<REPOSYNC>
"""

Because otherwise it could fail even without any packages in the output.

Reply:
Updated.